### DISCUSSION TITLE
[HxGrid] Drag and Drop Rows in Grid

### DISCUSSION THREAD

**Discussion by econner20** [havit/Havit.Blazor#765] [Q&A]
Are there any plans to add drag and drop rows in a grid to resort the row order such as the following:

https://blazor.syncfusion.com/demos/datagrid/draganddrop-within-grid?theme=bootstrap4

**Comment by hakenr** [comment]
Hello,

This feature is not currently on our roadmap.
(If you need it for your project, we can develop it for you on a commercial basis, provided you cover the development costs.)
